src/tests/ftest/launch.py

```diff
@@ -2260,6 +2309,18 @@ def main():
         action="store",
         help="comma-separated list of hosts to use as replacement values for "
              "client placeholders in each test's yaml file")
+    parser.add_argument(
+        "-th", "--logs_threshold",
+        action="store",
+        help="collect log sizes and report log sizes that go past provided"
+             "threshold. e.g. '-th 5M'"
+             "Valid threshold units are: B, K, M, G, T")
+    parser.add_argument(
+        "-tm", "--timeout_multiplier",
+        action="store",
+        default=None,
+        type=float,
+        help="a multiplier to apply to each timeout value found in the test yaml")
```

Comment:
Minor - Is `action="store"` necessary when both `default` and `type` are supplied?